### PR TITLE
demo page bugfixes

### PR DESCRIPTION
**Pull request by lukeselden**

### CODE REVIEW

**Review by ibirman-vbrick — APPROVED** [review]

File: demo/demo.js [hunk 1]
```diff
@@ -62,7 +62,9 @@ export function init(formDefaults, render) {
 function getConfig(formData) {
 	const {
 		config,
-		tokenValue, tokenType, tokenIssuer,
+		tokenValue: value,
+		tokenType: type,
+		issuer,
```
Comment (vbrickPMitchell) [inline demo/demo.js:67]: formData is the form fields. they should have a tokenValue/tokenType from the input[name] attrs.

File: demo/demo.js [hunk 2]
```diff
@@ -71,11 +73,7 @@ function getConfig(formData) {
 		...data,
 		config: {
 			...tryParse(config),
-			token: tokenValue && {
-				type: tokenType,
-				value: tokenValue,
-				issuer: tokenIssuer
-			}
+			token: value && { type, value, issuer }
```
Comment (vbrickPMitchell) [inline demo/demo.js:76]: roll this back like it was before.

File: demo/demo.js [hunk 3]
```diff
@@ -255,7 +259,7 @@ export function parseRevUrl(url) {
 
 	if(result.webcastId) {
 		result.tokenType = 'JWT';
-		result.tokenIssuer = 'vbrick_rev';
+		result.issuer = 'vbrick_rev';
```
Comment (vbrickPMitchell) [inline demo/demo.js:262]: tokenIssuer was correct

File: demo/index.js [hunk 4]
```diff
@@ -47,6 +47,7 @@ function embedContent(payload) {
 		? revSdk.embedVideo('#embed', videoId, embedConfig)
 		: revSdk.embedWebcast('#embed', webcastId, embedConfig);
 
+	globalThis.vbrickPlayer = currentEmbed;
```
Comment (vbrickPMitchell) [inline demo/index.js:50]: its not a player. 
vbrickEmbed?

**Review by vbrickPMitchell — CHANGES_REQUESTED** [review]

**Review by vbrickPMitchell — APPROVED** [review]

**Review by lukeselden — COMMENTED** [review]
changed per comments

**Pull request merged**